environment/src/main/java/com/sequenceiq/environment/parameters/dao/domain/AzureParameters.java

```diff
@@ -71,4 +84,12 @@ public String getEncryptionKeyUrlSecret() {
     public void setEncryptionKeyUrl(String encryptionKeyUrl) {
         this.encryptionKeyUrl = new Secret(encryptionKeyUrl);
     }
+
```

Comment:
Note: There is no getter/setter for `encryptionKeyUrlDummy` but that is probably OK if Hibernate does not need them. (I think it will access fields directly.)